boards/arm/nrf21540dk_nrf52840/nrf21540dk_nrf52840.dts

```diff
@@ -166,6 +166,7 @@ arduino_serial: &uart1 {
 	status = "okay";
 	current-speed = <115200>;
 	rx-pin = <33>;
+	rx-pull-up;
```

Comment:
it should also be added to `uart0` which is connected to segger chip but becomes floating after flipping `nRF only` switch. Original fix was due to increased power consumption after flipping this switch.

Reply:
Updated.